app/assets/stylesheets/steps.scss

```diff
@@ -121,3 +122,39 @@ $beige: #8a6d3b;
   padding: 10px 20px;
   margin: 20px 0 20px;
 }
+
+.add-change-note--details {
+  margin-bottom: 1em;
+
+  &[open] {
+    .add-change-note--summary {
+      display: none;
+    }
+  }
+
+  .add-change-note--summary::-webkit-details-marker {
+    display: none;
+  }
+}
+
+.change-note--details {
+  border: 1px solid $border-grey;
+  border-radius: 4px;
+  padding: .5em .5em 0;
```

Comment:
Just wondering why the switch from `px` to `em`? Most of our other padding and margin values are using `px`